### ISSUE TITLE
Can't compile the project, unless…

### ISSUE THREAD

**Issue by cmittendorf** [below/xkcd-Clock#1]
Hi!

I tried compiling this project but I always got an error that the header, that Xcode automatically generates, would be missing. Trying to build the framework alone didn't solve this problem (Xcode 6.2 and 6.3-beta), even then the error occurred.

```
error: 'XCWatchFace-Swift.h' file not found
#import "XCWatchFace-Swift.h"
        ^
<unknown>:0: error: could not build Objective-C module 'XCWatchFace'
```

However, after deleting the line `#import "XCWatchFace-Swift.h"`from `XCWatchFace.h` all parts of the project compiled successfully and I was able to run the app and  watch app. Pretty strange.

Christian

**Comment by below** [comment]
Yes, I have seen that issue. My solution was to build the Framework first (even though it is a dependency of the main target), then the file would be found. It was one of the things that made me sigh on twitter how broken the development tools are :(
